### PR TITLE
Pull request for code review

### PR DESCRIPTION
**Pull request by visva-dev**
## Microverse 6th project - Building with responsive design
- This project consists of replicating the  "TheNextWeb" website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.
- Aligned all elements with float and flex or grid.
- Used a mobile-first approach.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes:recycle: :traffic_light: 

Hi, @visva-dev and @ahmadchata, great job with this project :+1: PR looks professional, nice commit messages except that they are written in the past tense. Great use of the Mobile-first design approach. :smile: Great use of HTML semantics. Let us improve our project and get our project approved by working on the comments below: :slightly_smiling_face: 
- The larger image of the 3 articles should take 2/3 if the width
bigger
```Clone```
mmain1
```Source```
mmain
- Let us add a detailed description to the README file
- Let us use CSS float property for aligning elements for this project. Currently, we are missing this property
imp

Kindly implement these changes and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by ahmadchata** [comment]
## Changes Made

- The larger image in the article section now takes 2/3 of the width.
- A detailed description of the project is added to the README file.
- The screenshot in the README file is updated to the latest version.

**Comment by BrittanyBlake** [comment]
### Pull Request under review by Brittany Blake (TSE)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+
+  .news3 {
+    grid-area: news3;
+  }
+
+  .news-holder {
+    display: grid;
+    grid-template-areas:
+      'news news news2'
+      'news news news3';
```
Comment (BrittanyBlake) [inline styles.css]: Please use this  [article](https://developer.mozilla.org/es/docs/Web/CSS/grid-template-areas)  to determine the proper way to set the grid

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @ahmadchata and @visva-dev! 👋 👋 
Great job implementing the changes requested by the previous TSE! 🎖️ You have done a wonderful job so far! 👍 However, there is still one issue that needs to be addressed before you can move on 🔽
****

- [ ] As requested by the previous TSE, the larger image of the 3 articles should take 2/3 if the width. Your clone still does not satisfy this requirement:
Original Page 👇 
Screen Shot 2020-06-06 at 4 03 22 PM
Your clone 👇 
Screen Shot 2020-06-06 at 4 03 34 PM

You can find more information about how to use `grid-template-area` [here](https://developer.mozilla.org/es/docs/Web/CSS/grid-template-areas)
****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by ahmadchata** [comment]
## Changes Made

- The larger image in the article section now takes 2/3 of the width for 768px

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

_P.S. Make sure to update the live demo link in your README file_ 😄 

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**